docs/tutorials/deploy_sample_app.md

```diff
@@ -60,158 +30,67 @@ To verify that `dfx` properly installed, run:
-
-For this, we recommend using two terminals:
-
--   **Terminal A** shows the output of the execution environment. This is analogous to starting a local server in a Web2 project, e.g. node.js’s Express, python’s Django, or Ruby’s Rails.
-
--   **Terminal B** is used to interact with the canister running in the execution environment. This is analogous to sending HTTP API messages to servers running locally, e.g. Postman or Panic.
-
-To distinguish the two terminals in this tutorial, terminal A has a dark blue background...
-
-![dfx new](_attachments/dfx-new-hello-2.png)
-
-... and terminal B has a black background.
-
-![terminal b ls](_attachments/terminal-b-ls.png)
-
-### Start the execution environment
-
-<!-- Navigate to the root directory for your project, if necessary. In this tutorial, you should be in the folder `hello` because that is the name of the project created in section 2 above.
-
-Start the local canister execution environment in Terminal A: -->
-
-In terminal A, navigate to the root directory `hello` of our project and run
-
-``` bash
 dfx start
 ```
 
-![dfx start](_attachments/terminal-a-dfx-start.png) 
+-   The `dfx new hello` command created a new project called `hello` in your local machine
+-   `cd hello` navigates your terminal into the newly-created `hello` project
+-   `dfx start` starts a local instance of the IC to test and deploy too
 
 :::note
 
 -   Depending on your platform and local security settings, you might see a warning displayed. If you are prompted to allow or deny incoming network connections, click "Allow."
--   Check no other network process is running that would create a port conflict on 8000.
 
 :::
 
-**Congratulations - there is now a local Instance of the execution environment of the IC running on your machine! Leave this window/tab open and running while you continue.** If the window/tab is closed, the local instance of the execution environment of the IC will not be running and the rest of the tutorial will fail.
-
-### Deploy the dapp locally
+**🎉 Congratulations! 🎉 **
 
-:::note
-Since this is only a canister execution environment, this deployment has fewer steps than a deployment to mainnet, which requires [cycles](/concepts/tokens-cycles.md)).
-:::
+There is now a local Instance of the execution environment of the IC running on your machine! Leave this window/tab running while you continue so the tutorial can continue without issues.
 
-<!-- To deploy your first dapp locally:
+## 3. Deploy the dapp locally
 
-#### 3.2.1 Check that you are still in the root directory for your project, if needed.
-
-Ensure that node modules are available in your project directory, if needed, by running the following command (it does not hurt to run this many times): -->
-
-In terminal B, navigate to the root directory `hello` of our project. Install all the necessary node modules by running
+In a new terminal window, navigate to the root directory `hello` project, then run:
```

Comment:
```suggestion
Open a new terminal window, navigate to the root directory `hello` project, and then run:
```
I think it's too easy to skim over the need for two windows with this phrasing.

Alternatively: @ericswanson-dfinity Do we still need to advocate for the two-window approach? I don't think I've ever used `dfx start` instead of `dfx start --background` once this year.

Reply:
Good point. Maybe it should be `dfx start --background`